data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst

```diff
@@ -102,7 +102,7 @@ Greater Power ECL Boost					KEY:Feyri ~ Greater Power ECL Boost		VISIBLE:NO
 
 
 ###Block:
-Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	BONUS:VAR|LevelAdjustment|-1
+Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	PRERACE:1,Planetouched (Aasimar),Planetouched (Tiefling)	BONUS:VAR|LevelAdjustment|-1
```

Comment:
There, that should be it now.

On 14 January 2016 at 18:52, Dave Griffin <ferret.griffin@googlemail.com>
wrote:

> I guess yes, I'll add those now. Have just fixed Turn vars on Cloistered
> Cleric so will re-commit shortly.
>
> On 14 January 2016 at 18:50, Bahamut <notifications@github.com> wrote:
>
>> In
>> data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/players_guide_to_faerun/pg_templates.lst
>> <https://github.com/BahamutDragon/pcgen/pull/354#discussion_r49767111>:
>>
>> > @@ -102,7 +102,7 @@ Greater Power ECL Boost					KEY:Feyri ~ Greater Power ECL Boost		VISIBLE:NO
>> >
>> >
>> >  ###Block:
>> > -Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	BONUS:VAR|LevelAdjustment|-1
>> > +Lesser Planetouched		VISIBLE:YES		BONUS:VAR|DefaultRaceType|1	ABILITY:Internal|AUTOMATIC|Assign Race Type ~ Humanoid	TYPE:Humanoid	SUBRACE:Lesser	PRERACE:1,Planetouched (Aasimar),Planetouched (Tiefling)	BONUS:VAR|LevelAdjustment|-1
>>
>> I have Lesser Genasi coded up as well, should they be added to the
>> PRERACE?
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/BahamutDragon/pcgen/pull/354/files#r49767111>.
>>
>
>